synop2bufr/__init__.py

```diff
@@ -228,32 +232,38 @@ def parse_synop(message: str, year: int, month: int) -> dict:
             output['cloud_cover'] = None
 
     # Wind direction is already in degrees
-    if 'surface_wind' in decoded:
+    if ('surface_wind' in decoded) and ENCODE_WIND:
```

Comment:
What happens when iw is not 1 or 4 (and so ENCODE_WIND if false)?

Reply:
In the new 307096 template, there is no longer a key for the instrumentation for wind measurement. Enrico and I believe that is because there is only one way to measure the wind: with an anemometer. The only other options for iw are 0 and 3, which indicates the wind measurement has been estimated. We decided in this case the corresponding wind data should not be encoded.

Reply:
Following up on discussion with @RoryPTB and @efucile, we want to make sure we still exchange the visual winds as excluding these is not a decision we can make without consultation (e.g. see https://gcos.wmo.int/en/essential-climate-variables/about/gcos-monitoring-principles).

As a compromise, we want to use the SYNOP BUFR sequence (307080) when the winds are visually reported and the recommended GBON sequence (307096) otherwise.